App/PHP_HTML/login sample/signIn_1.php

```diff
@@ -48,8 +48,16 @@
         <input required id="password" type="password" name="user_pass" size="48" placeholder="Password"><br><br>
         <br>
         <div style="padding-bottom: 25px;"><input style="display: block; margin:0 auto;" id="sub" type="submit" name="submitform" value="Log In"></div>
+        <a href="forgotPassword.php">Forgot password?</a>
+
+        <br>  <br>
     </form>
-  
+    <?php if("localhost/SOEN341-SA1/App/PHP_HTML/login%20sample/signIn_1.php?problem=Error!IncorrectPassword." == $_SERVER['SERVER_NAME'].$_SERVER['REQUEST_URI'])
```

Comment:
The best practice here is to use

if(isset($_GET["problem"]){
   echo $_GET["problem"];  //with the previous styling from master
}


An absolute path is to be avoided as it might not work on other dev environement and will not work on AWS